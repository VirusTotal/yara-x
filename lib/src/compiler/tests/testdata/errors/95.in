rule test {
   strings:
     $a = "foo"
     $_b = "bar"  
     $c = "baz"
   condition:
     $a
}