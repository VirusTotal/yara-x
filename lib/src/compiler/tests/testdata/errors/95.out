error: unused pattern `$c`
 --> line:5:6
  |
5 |      $c = "baz"
  |      ^^ this pattern was not used in the condition
  |